server/datastore/mysql/testing_utils.go

```diff
@@ -22,33 +23,30 @@ const (
 	testAddress  = "localhost:3307"
 )
 
-func connectMySQL(t *testing.T, testName string) *Datastore {
-	config := config.MysqlConfig{
-		Username: testUsername,
-		Password: testPassword,
-		Database: testName,
-		Address:  testAddress,
-	}
+func init() {
+	// Initialize the schema once for the entire test run.
+	initializeSchemaOrPanic()
+}
 
-	// Create datastore client
-	ds, err := New(config, clock.NewMockClock(), Logger(log.NewNopLogger()), LimitAttempts(1))
-	require.Nil(t, err)
-	return ds
+func panicif(err error) {
```

Comment:
This reminds me a bit of the [go2 errors proposal](https://go.googlesource.com/proposal/+/master/design/go2draft-error-handling.md) :)

Reply:
I thought we would've had that solved after the few years I hadn't touched go... but here we are.